r/R/dplyr-functions.R

```diff
@@ -250,6 +250,39 @@ nse_funcs$is_logical <- function(x, n = NULL) {
   nse_funcs$is.logical(x)
 }
 
+# Create a data frame/tibble/struct column
+nse_funcs$tibble <- function(..., .rows = NULL, .name_repair = NULL) {
+  if (!is.null(.rows)) arrow_not_supported(".rows")
+  if (!is.null(.name_repair)) arrow_not_supported(".name_repair")
```

Comment:
Would it be possible (and desirable) to have these warn + ignore them instead of erroring? Most of them are checking processes and don't change the contents of the data.frame/tibble drastically anyway.

Reply:
I agree with erroring if someone provides these arguments